src/odemis/driver/test/technolution_test.py

```diff
@@ -495,8 +495,8 @@ def test_resolution_VA(self):
         self.EBeamScanner.resolution.value = (7 * min_res, 3 * min_res)
         self.assertEqual(self.EBeamScanner.resolution.value, (7 * min_res, 3 * min_res))
 
-        # Check if values which do not have effective cell size (= resolution/shape) with a multiple of 4 the
-        # the resolution gets changed correcctly.
+        # Check if for a requested resolution values, where the effective cell size is not a multiple of 4, the closest
```

Comment:
"a" and the plural of "values" exclude each other ;)
`Check if for a requested resolution value, where....`